pkgs/tools/misc/nncp/default.nix

```diff
@@ -9,24 +10,28 @@
 
 stdenv.mkDerivation rec {
   pname = "nncp";
-  version = "5.3.3";
+  version = "5.4.1";
 
   src = fetchurl {
     url = "http://www.nncpgo.org/download/${pname}-${version}.tar.xz";
-    sha256 = "1l35ndzrvpfim29jn1p0bwmc8w892z44nsrdnay28k229r9dhz3h";
+    sha256 = "0qzs4zkaj6qmg7ghqab4ba07p7w5qkw05q8pbh8zhgaka6dhqbd0";
   };
 
-  nativeBuildInputs = [ go ];
+  nativeBuildInputs = [ go redo-apenwarr ];
 
-  preConfigure = ''
+  buildPhase = ''
     export GOCACHE=$PWD/.cache
+    export CFGPATH=/etc/nncp.hjson
+    export SENDMAIL=/run/wrappers/bin/sendmail
```

Comment:
Shouldn't this point to the nix version or am I missing something?

Reply:
It's only a default value, not a dependency. Do you have a better suggestion? What would be the nix version, without adding a dependency?

Reply:
You can't add a default enabled value without adding a depedency.

Reply:
what's the solution?
- keep upstream /usr/sbin/sendmail, or
- set it to /run/wrappers/bin/sendmail which only works on NixOS, or
- pull in a not-needed and maybe bloated and unconfigured MTA dependency?

Also I must note this is not the main topic of this PR and the path is already there.